templates/hello-world/flake.nix

```diff
@@ -87,7 +87,7 @@
       # Package a virtual environment as our main application.
       #
       # Enable no optional dependencies for production build.
-      packages.x86_64-linux.default = pythonSet.mkVirtualEnv "hello-world-env" workspace.deps.default;
+      packages.x86_64-linux.default = pythonSet.mkVirtualEnv "hello" workspace.deps.default;
```

Comment:
Just changing this to `"hello"` makes it not very descriptive. I'd rather have `nix run` not work and have a description rather than just `"hello"`.
It's really annoying that the flakes cli makes these assumptions about the package layout based on a derivation name.